addons/voice/org.openhab.voice.googletts/ESH-INF/config/config.xml

```diff
@@ -27,6 +27,11 @@
 			<description>Speaking rate can be 4x faster or slower than the normal rate</description>
 			<default>1</default>
 		</parameter>
+		<parameter name="purgeCache" type="boolean">
+			<label>Purge the cache</label>
+			<description>Purge the cache after configuration changes</description>
```

Comment:
I think it should be emphasized here that after enabling it, users also need to disable it again. Otherwise users change it to true, continue with their life and after restarting openHAB their caches are always purged. 

It would have been nice if users could just press a button to purge the cache. But since ESH only allows for changing configuration values this is not possible.

Another option would be to change the setting back to false from within the binding after having purged the caches. E.g. using the OSGi `ConfigurationAdmin` service. Though I do not see any other extension change their own settings with it. So that could introduce an architectural issue.

Reply:
I'd rather also have this added to the configuration parameter description because in my own experience people can't find or don't read the documentation. :wink: 

After looking at the implementation I'd rephrase the description to:

> Purges the cache e.g. after testing different voice configuration parameters. When enabled the cache is purged once. Make sure to disable this setting again so the cache is maintained after restarts.

Reply:
Description added